_apps/prr-review-counter.md

```diff
@@ -0,0 +1,19 @@
+---
+# A human-friendly name of your listing
+title: Pull Request Review Counter
+# A short description of what your app does
+description: Require multiple approved reviews before merging a Pull Request.
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: pull-request-review-counter
+# Include a few screenshots that show your app in action
+screenshots:
+- https://user-images.githubusercontent.com/82819/37225532-c290047e-238b-11e8-9443-0322d16cc0b1.png
+# The GitHub usernames of anyone who authored the app
+authors: [ gcschmit, rycao18, saanvid, helenagao14 ]
+# The repository where the code is located
+repository: probotics2018/PRRCounter
+# The address where this app is deployed
+host: https://github.com/apps/pull-request-review-counter
+---
+
+Pull Request Review Counter ensures that all PRs in your repository can only be merged after X number of reviewers with write access have approved the PR.
```

Comment:
How is the app configured? It'd be great to include docs for that here.

Reply:
@Phanatic @bkeepers The app relies on a file being in the master branch of the repository, which should be called `Github.json` , which should be structured like this 

`{
	"approved-reviewer-count": 0
}
`